org.srb2.SRB2Kart.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: org.srb2.SRB2Kart
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: srb2kart
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.srb2kart
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: game-music-emu
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+      - /lib/*.so
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz
+        sha256: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d
+
+  - name: srb2kart
+    buildsystem: simple
+    build-options:
+      arch:
+        arm:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX=1 NONX86=1'
+        aarch64:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX64=1'
+        x86_64:
+          env:
+            ARCH_MAKE_ARGS: 'LINUX64=1 X86_64=1'
+    build-commands:
+      - make -C src -j $FLATPAK_BUILDER_N_JOBS NOUPX=1 SDL=1 $ARCH_MAKE_ARGS
+      - install -D -m 755 -t $FLATPAK_DEST/bin bin/Linux*/Release/lsdl2srb2kart
+      - install -D -m 644 srb2.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -D -m 755 srb2kart.sh $FLATPAK_DEST/bin/srb2kart
+      - install -D -m 644 -t $FLATPAK_DEST/share/applications $FLATPAK_ID.desktop
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/STJr/Kart-Public
+        tag: v1.2
+        commit: f856f18233da492c9446f70a2856e2e178ddfd1a
+      - type: script
+        commands:
+          - ls ~/.srb2kart/srb2.srb || zenity --error --title "Missing game assets" --text "Game assets not found, please add them to your ~/.var/app/org.srb2.SRB2Kart/.srb2kart folder. See https://github.com/STJr/Kart-Public/releases for download instructions." --ok-label "Close" --width=400
```

Comment:
I assume this is not needed anymore?

Reply:
its not!  I thought I took that out, but maybe that was on a different branch

Reply:
It was a different branch.  @ghisvail can you merge the `no_zen` pull request?